discord.user.css

```diff
@@ -0,0 +1,437 @@
+    }
+    else if (theme=="Mocha") {
+        $type = dark
+    
+        $rosewater = #f5e0dc
+        $flamingo = #f2cdcd
+        $pink = #f5c2e7
+        $mauve = #cba6f7
+        $red = #f38ba8
+        $maroon = #eba0ac
+        $peach = #fab387
+        $yellow = #f9e2af
+        $green = #a6e3a1
+        $teal = #94e2d5
+        $sky = #89dceb
+        $sapphire = #74c7ec
+        $lavender = #b4befe
+        $blue = #89b4fa
+        $text = #cdd6f4
+        $subtext1 = #bac2de
+        $subtext0 = #a6adc8
+        $overlay2 = #9399b2
+        $overlay1 = #7f849c
+        $overlay0 = #6c7086
+        $surface2 = #585b70
+        $surface1 = #45475a
+        $surface0 = #313244
+        $base = #1e1e2e
+        $mantle = #181825
+        $crust = #11111b
+    }
+    
+    .theme-dark,
+    .theme-light,{
+        --brand-experiment: $rosewater; // accent color
+        --brand-experiment-560: $rosewater;
+        --brand-experiment-200: $base; // Mention text
+        --brand-experiment-30a: $rosewater; // Mention background
+        --brand-experiment-15a: $surface2;
+        --brand-experiment-600: selectedButtonBG($rosewater);
+            
+        if (usefont) {
+            --font-primary: Inter,Whitney,"Helvetica Neue",Helvetica,Arial,sans-serif;
+            --font-display: Inter,"ABC Ginto Normal","Helvetica Neue",Helvetica,Arial,sans-serif;
+            --font-headline: Inter,"ABC Ginto Nord","Helvetica Neue",Helvetica,Arial,sans-serif;
+            --font-code: Consolas,"Andale Mono WT","Andale Mono","Lucida Console","Lucida Sans Typewriter","DejaVu Sans Mono","Bitstream Vera Sans Mono","Liberation Mono","Nimbus Mono L",Monaco,"Courier New",Courier,monospace;
+        }
+        
+        --header-primary: $subtext0;
+        --header-secondary: $subtext1;
+        --text-normal: $text;
+        --text-muted: $subtext0;
+        --text-link: classicLinks ? rgb(0, 175, 244) : $rosewater;
+        //--text-link-low-saturation: hsl(197,calc(var(--saturation-factor, 1)*100%),52.9%);
+        --text-positive: $green;
+        --text-warning: $yellow;
+        --text-danger: $red;
+        --text-brand: $blue;
+        --interactive-normal: $overlay2;
+        --interactive-hover: $overlay2;
+        --interactive-active: $text;
+        --interactive-muted: $surface1;
+        --background-primary: $base;
+        --background-secondary: $mantle;
+        --background-secondary-alt: $crust;
+        --background-tertiary: $crust;
+        --background-accent: $surface1;
+        --background-floating: $mantle;
+        --background-nested-floating: #2f3136;
+        --background-mobile-primary: #36393f;
+        --background-mobile-secondary: #2f3136;
+        --chat-background: #36393f;
+        --chat-border: #202225;
+        --chat-input-container-background: #36393f;
+        --background-modifier-hover: $surface1;
+        --background-modifier-active: $surface2;
+        --background-modifier-selected: $surface2;
```

Comment:
Suggestions: Use Surface0 for selected elements for a better background - element - foreground contrast. c:

Reply:
I'll make these modifications myself after merging it into the dev branch.

Reply:
Fixed in 00ea5f0